libafl_qemu/src/qemu/mod.rs

```diff
@@ -57,6 +57,18 @@ pub enum QemuError {
     RW(QemuRWError),
 }
 
+impl From<QemuError> for libafl::Error {
+    fn from(qemu_error: QemuError) -> Self {
+        libafl::Error::unknown(qemu_error)
```

Comment:
There are things like `TooManyArgs` that sound like they map to `Error::illegal_argument` for examle